src/views/search/results.njk

```diff
@@ -19,50 +19,46 @@
     Test centres near '{{ searchNormalised | escape }}'
     <p class="govuk-body-s"></p>
   </h1>
-  <div class="govuk-grid-row">
-    <div class="govuk-grid-column-one-half govuk-hint">
-      {% set start = (paginationSettings.currentPage - 1) * paginationSettings.perPage + 1 %}
-      {% set end = start + paginationSettings.itemsCount - 1 %}
-      {% set total = paginationSettings.scannedItemsCount %}
-      Showing {{ start }} to {{ end }} of {{ total }} results
-    </div>
-    <div class="govuk-grid-column-one-half">
-      {% set filterUrl = "?postcode=" + search %}
-      <form action={{filterUrl}} method="POST" id="filter-results-form">
-        {{ govukCheckboxes({
-          name: "filters",
-          classes: "govuk-checkboxes--small",
-          fieldset: {
-            legend: {
-              text: "Filter the results"
-            }
-          },
-          items: [
-            {
-              id: "remove-no-availability-filter-checkbox",
-              value: "removeNoAvailability",
-              text: "Remove fully booked test centres",
-              checked: filters.removeAtfsWithNoAvailability
-            }
-          ]
-        }) }}
-        <div class="govuk-button-group">
-          {{ govukButton({
-            text: "Filter",
-            attributes: {
-              id: "filters-submit-button"
-            }
-          }) }}
+  <div class="govuk-hint">
+    {% set start = (paginationSettings.currentPage - 1) * paginationSettings.perPage + 1 %}
+    {% set end = start + paginationSettings.itemsCount - 1 %}
+    {% set total = paginationSettings.scannedItemsCount %}
+    Showing {{ start }} to {{ end }} of {{ total }} results
+  </div>
+  {% set filterUrl = "?postcode=" + search %}
+  <form action={{filterUrl}} method="POST" id="filter-results-form">
 
-          {% set clearFiltersLink = "?postcode=" + search %}
-          <a class="govuk-link govuk-link--no-visited-state govuk-!-padding-1"
-            href={{clearFiltersLink}} id="clear-filters-link">
-            Clear filters
-          </a>
+    <div class="govuk-form-group dvsa-no-margin-bottom">
+      <fieldset class="govuk-fieldset">
+        <legend class="govuk-fieldset__legend">
+          Show:
+        </legend>
+        <div class="govuk-radios govuk-radios govuk-radios--small govuk-radios--inline">
+            <div class="govuk-radios__item">
+              <input class="govuk-radios__input" id="clear-filters-radio" name="filters" type="radio"
```

Comment:
I would have set the id as show-all-results-radio-button because in theory you would be displaying all the results when this is selected